script.js

```diff
@@ -0,0 +1,100 @@
+// A user should be able to select numbers so that they can do things with them.
+// A user should be able to add numbers together.
+// A user should be able to subtract numbers.
+// A user should be able to divide numbers.
+// A user should be able to multiply numbers together.
+// A user should be able to see the output.
+//event.currentTarget
+
+const display = document.getElementById("calc-display");
+console.log(display);
+//const buttons = document.querySelector("[class=number-button]");
+const buttons = document.querySelectorAll("button");
+
+console.log(buttons);
+
+let calculation = [];
+let result;
+
+function calculate(button) {
+  const value = button.textContent;
+
+  if (value == "Reset") {
+    calculation = [];
+    display.textContent = ".";
+  } else if (value === "=") {
+    display.textContent = eval(result);
+  } else {
+  }
+}
+
+buttons.forEach((button) =>
+  button.addEventListener("click", () => calculate(button))
```

Comment:
the event listener here is setup correctly, but the problem is that you are not really workign with the trigger of the event instead you are trying to operate on the button you grabbed inside the array. 
always try to work with the event target not the reference to the selected element. 

`"click", (event) => calculat(event.target))`

also, i would try to break it down into steps, why bothjer evaluating anything if you dont know if a value is being stored or accessed, start slow, just print the number into the console when you click a button, then work your way up. no point to reset the value if you cant get a value to start. 

also adding calculation intgo an array is ok, but may complicate your solution in the future, just start simple, one var for each number val in the equation, one var for the operation